app-admin/passwordsafe/passwordsafe-1.19.1.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WX_GTK_VER="3.2-gtk3"
+
+inherit cmake desktop optfeature wxwidgets xdg
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/pwsafe-${MY_PV}"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="qr test +xml yubikey"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	net-misc/curl
+	sys-apps/util-linux
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+BDEPEND="
+	app-arch/zip
+	dev-lang/perl
+	sys-devel/gettext
+	virtual/pkgconfig
+	test? ( dev-cpp/gtest )"
+
+src_configure() {
+	setup-wxwidgets
+
+	local mycmakeargs=(
+		-DNO_QR=$(usex !qr)
+		-DNO_GTEST=$(usex !test)
+		-DGTEST_BUILD=OFF
+		-DXML_XERCESC=$(usex xml)
+		-DNO_YUBI=$(usex !yubikey)
+	)
+
+	cmake_src_configure
+}
+
+src_install() {
+	pushd "${BUILD_DIR}" || die
+
+	dobin pwsafe
+	dobin cli/pwsafe-cli
+	dosym pwsafe /usr/bin/${PN}
+	dosym pwsafe-cli /usr/bin/${PN}-cli
+
+	insinto /usr/share/locale
+	doins -r src/ui/wxWidgets/I18N/mos/*
+
+	insinto /usr/share/${PN}/help
+	doins help/*.zip
+
+	popd || die
+
+	newman docs/pwsafe.1 ${PN}.1
+
+	dodoc README.md README.LINUX.* SECURITY.md docs/{ReleaseNotes.md,ChangeLog.txt}
+
+	insinto /usr/share/${PN}
+	doins -r xml
+
+	doicon -s 48 install/graphics/48x48/pwsafe.png
+	doicon -s 128 install/graphics/128x128/pwsafe.png
+	newmenu install/desktop/pwsafe.desktop ${PN}.desktop
```

Comment:
I know that this is just a bump to an existing package, but I was curious why this package is doing this at all. As far as I can tell, at least nowadays it shouldn't since upstream has a mostly usable installation routine.

Although it could really use a patch upstream to stop compressing the manpage, as *no* distro wants this. Every distro either (re)compresses manpages with their preferred compression, uncompresses it because they don't want compression at all, or raises some form of QA warning objecting to the presence of non package manager induced compression.

This could simplify things and mean less adapting to upstream changes, e.g. this bump introduces:

```diff
diff --git a/app-admin/passwordsafe/passwordsafe-1.18.2.ebuild b/app-admin/passwordsafe/passwordsafe-1.19.1.ebuild
similarity index 93%
copy from app-admin/passwordsafe/passwordsafe-1.18.2.ebuild
copy to app-admin/passwordsafe/passwordsafe-1.19.1.ebuild
index 9581dc5e6d16..aba4b1f16b18 100644
--- a/app-admin/passwordsafe/passwordsafe-1.18.2.ebuild
+++ b/app-admin/passwordsafe/passwordsafe-1.19.1.ebuild
@@ -15,7 +15,7 @@ S="${WORKDIR}/pwsafe-${MY_PV}"
 
 LICENSE="Artistic-2"
 SLOT="0"
-KEYWORDS="amd64 ~arm64 ~x86"
+KEYWORDS="~amd64 ~arm64 ~x86"
 IUSE="qr test +xml yubikey"
 RESTRICT="!test? ( test )"
 
@@ -74,7 +74,8 @@ src_install() {
 	insinto /usr/share/${PN}
 	doins -r xml
 
-	doicon -s 48 install/graphics/pwsafe.png
+	doicon -s 48 install/graphics/48x48/pwsafe.png
+	doicon -s 128 install/graphics/128x128/pwsafe.png
 	newmenu install/desktop/pwsafe.desktop ${PN}.desktop
 }
```

Reply:
@eli-schwartz I took some time and tried to achieve the same installation result with the upstream install method.
I added a small patch for the makefile, and re-posted the revised ebuild. Maybe you may take a look again?